sources/xk.json

```diff
@@ -11,5 +11,18 @@
             "info@openaq.org"
         ],
         "active": true
+    },
+    {
+        "url": "http://kosovo-airquality.com/secure/ValueTable.html",
+        "adapter": "kosovo",
+        "name": "Kosovo",
+        "city": "",
+        "country": "XK",
+        "description": "Kosovo data from kosovo-airquality.com",
+        "sourceURL": "http://kosovo-airquality.com/secure/index2.html",
+        "contacts": [
+            "info@openaq.org"
```

Comment:
This is a mostly unused field at the moment, but in the future, we may thank contributors or send notifications if they want to know when a source isn't working. With that in mind, feel free to add your email here if you want. Also no worries if not!